Eos/Fuego.H

```diff
@@ -165,6 +165,12 @@ struct Fuego
     // CKWYR(&R, &T, Y, WDOT);
     amrex::Real C[NUM_SPECIES];
     CKYTCR(&R, &T, Y, C);
+
+    // Do not allow negative C for wdot calculation
+    for (int ii = 0; ii < NUM_SPECIES; ++ii) {
```

Comment:
I understand this fix is in the lower level fuego routines but can there be flag to turn this on and off by the user?

Reply:
Based on the above discussion showing slowdowns with this change,  I agree making this a runtime option would be better. But with the way the EOS code is presently set up, that would be hard to do. There's no way to pass input file parameters to EOS calls right now (e.g. even for GammaLaw EOS, Gamma is hardcoded).  

So I don't think this PR should be integrated, at least for now. It may be a component of robustification, but I think it needs to wait until we better understand what's going on.